ldaptive/src/main/java/org/ccci/idm/user/ldaptive/dao/io/GroupValueTranscoder.java

```diff
@@ -1,96 +1,68 @@
 package org.ccci.idm.user.ldaptive.dao.io;
 
-import com.google.common.base.Strings;
```

Comment:
`GroupValueTranscoder` has now almost become a glorified `DnUtils` with `ValueTranscoder` interface. I suppose that interface is helpful for those API's that expect it.